src/content/contributing/developers/_index.en.md

```diff
@@ -1,8 +0,0 @@
----
-title: "Developers"
-date: 2020-03-18T16:03:26+01:00
-weight: 11
----
-
-Submariner strives to be an open, welcoming community for developers.
```

Comment:
I think this phrase was great, may be we can conserve it into the  src/content/contributing/building_testing/_index.en.md

:-)

Reply:
Thanks, good idea, done in the latest. :)